### ISSUE TITLE
Error output when typing -a with zsh-autosuggestions

### ISSUE THREAD

**Issue by koterpillar** [atuinsh/atuin#1690]
On a fresh Debian container, install Atuin and zsh-autosuggestions:

```
apt update && apt install -y build-essential git curl zsh
git clone https://github.com/zsh-users/zsh-autosuggestions ~/.zsh/zsh-autosuggestions
echo 'source ~/.zsh/zsh-autosuggestions/zsh-autosuggestions.zsh' > ~/.zshrc
/bin/bash -c "$(curl --proto '=https' --tlsv1.2 -sSf https://setup.atuin.sh)"
```

Then execute zsh (`exec zsh`) and type `-a` or `--a`.

Instead of expected behaviour (nothing, or previous commands starting with `-a` suggested) I get:

```
3a557c0a9e20# -aerror: unexpected argument '-a' found

  tip: to pass '-a' as a value, use '-- -a'

Usage: atuin search [OPTIONS] [QUERY]...

For more information, try '--help'.
```

I'm raising this with Atuin first because of [this answer](https://forum.atuin.sh/t/always-on-inline-autocomplete/137/2) implying zsh-autosuggestions is supported.

**Comment by akinomyoga** [comment]
I think this is a duplicate of #1582. It's fixed in the main branch (#1506) and will be released this week (#1685).

**Comment by koterpillar** [comment]
Apologies, I should have checked for duplicates.
